Nest - 00 Foreword (#3)
raven-os/getting-started: nest/00. Foreword.md

# Getting started with Nest
## Foreword
Welcome to Nest's official getting-started guide! This guide assumes you are using the latest version of Nest, which is 0.1.0 at the time of writing.

In order to avoid any confusion, we will refer to the project as a whole as `Nest` (with an upper case n), while we will refer to the binary as `nest` (with a lower case n). Thus, `finest` refer to its eponymous binary, and is included in Nest.

[//]: # (TODO: add link to the installation instructions page of Raven-OS)

> Nest is a dependency-based package manager for [Raven-OS](https://raven-os.org), which is an adaptive, customizable and evolutive open-source Operating System (OS) based on the Linux kernel. To use Nest, it's recommended to install Raven-OS to run Nest in it, because Nest may overlap with an already installed package manager. To install Raven-OS, follow the instructions [here](). Nest is included in the installation of Raven-OS by default, so you can start using it right away.

Reviewer: ```suggestion
Nest is a dependency-based package manager for [Raven-OS](https://raven-os.org), which is an adaptive, customizable and evolutive open-source Operating System (OS) based on the Linux kernel. To use Nest, it's recommended to install Raven-OS and to run Nest in it, because Nest may overlap with an already installed package manager. To install Raven-OS, follow the instructions [here](). Nest is included in the installation of Raven-OS by default, so you can start using it right away.
```

> Nest is a dependency-based package manager for [Raven-OS](https://raven-os.org), which is an adaptive, customizable and evolutive open-source Operating System (OS) based on the Linux kernel. To use Nest, it's recommended to install Raven-OS and to run Nest in it, because Nest may overlap with an already installed package manager. To install Raven-OS, follow the instructions [here](). Nest is included in the installation of Raven-OS by default, so you can start using it right away.

Reviewer 2: ```suggestion
Nest is a dependency-based package manager for [Raven-OS](https://raven-os.org), which is an adaptive, customizable and evolutive open-source Operating System (OS) based on the Linux kernel. To use Nest, it's recommended to install Raven-OS and to run Nest in it, because Nest may overlap with an already installed package manager if you try running it in your current Linux installation.
To install Raven-OS, follow the instructions [here](). Nest is by default included in the installation of Raven-OS, so you can start using it right away.
```

## How to read this guide

> nest is the binary targeted towards newcomers. If this is the first time you are using Raven-OS, or more generally a GNU/Linux distribution, it's strongly recommended that you use this binary instead of finest. The latter is targeted towards more experienced users, and has more advanced commands. Both are installed by default with Nest, so you don't need to worry about installing them separately.

Reviewer 2: Use `code text` when refering to binaries, I believe it accentuates even more the fact that you're talking about a binary and not the whole project. (Same for `finest`)

Author: good idea. I did it the first time I mentioned the two binaries, but I stopped doing it later for some obscure reason

[//]: # (TODO: Add link to the chapter 1)
[//]: # (TODO: Add link to the chapter 2)
[//]: # (TODO: Add link to the chapter 3)
[//]: # (TODO: Add link to the chapter 4)

> The [introduction]() details general but important concepts to understand to be able to use Nest.  The [chapter 2]() explains nest's commands. The [chapter 3]() explains some important concepts behind Nest. Although reading it is recommended, it's not required for the usage of nest (but it is for the usage of finest). The [chapter 4]() details the key concepts behind finest and its commands. In the [chapter 5](), you will find a cheat sheet, with all nest's and finest's commands, and a short explanation on what they do. Finally, to conclude this guide, you will find a glossary in the [chapter 6](). It links all the commands, the important concepts and the technical terms mentioned in this guide to their respective chapters. If you're searching for explanations related to Nest, you should search there.

Reviewer: ```suggestion
The [introduction]() details general but important concepts to understand to be able to use Nest. The [chapter 2]() explains nest's commands. The [chapter 3]() explains some important concepts behind Nest. Although reading it is recommended, it's not required for the usage of nest (but it is for the usage of finest). The [chapter 4]() details the key concepts behind finest and its commands. In the [chapter 5](), you will find a cheat sheet, with all nest's and finest's commands, and a short explanation on what they do. Finally, to conclude this guide, you will find a glossary in the [chapter 6](). It links all the commands, the important concepts and the technical terms mentioned in this guide to their respective chapters. If you're searching for explanations related to Nest, you should search there.
```
(fix double space)

> The [introduction]() details general but important concepts to understand to be able to use Nest.  The [chapter 2]() explains nest's commands. The [chapter 3]() explains some important concepts behind Nest. Although reading it is recommended, it's not required for the usage of nest (but it is for the usage of finest). The [chapter 4]() details the key concepts behind finest and its commands. In the [chapter 5](), you will find a cheat sheet, with all nest's and finest's commands, and a short explanation on what they do. Finally, to conclude this guide, you will find a glossary in the [chapter 6](). It links all the commands, the important concepts and the technical terms mentionned in this guide to their respective chapters. If you're searching for explanations related to Nest, you should search there.

Reviewer: ```suggestion
The [introduction]() details general but important concepts to understand to be able to use Nest.  The [chapter 2]() explains nest's commands. The [chapter 3]() explains some important concepts behind Nest. Although reading it is recommended, it's not required for the usage of nest (but it is for the usage of finest). The [chapter 4]() details the key concepts behind finest and its commands. In the [chapter 5](), you will find a cheat sheet, with all nest's and finest's commands, and a short explanation on what they do. Finally, to conclude this guide, you will find a glossary in the [chapter 6](). It links all the commands, the important concepts and the technical terms mentioned in this guide to their respective chapters. If you're searching for explanations related to Nest, you should search there.
```
mentionned -> mentioned

> The [introduction]() details general but important concepts to understand to be able to use Nest. The [chapter 2]() explains nest's commands. The [chapter 3]() explains some important concepts behind Nest. Although reading it is recommended, it's not required for the usage of nest (but it is for the usage of finest). The [chapter 4]() details the key concepts behind finest and its commands. In the [chapter 5](), you will find a cheat sheet, with all nest's and finest's commands, and a short explanation on what they do. Finally, to conclude this guide, you will find a glossary in the [chapter 6](). It links all the commands, the important concepts and the technical terms mentioned in this guide to their respective chapters. If you're searching for explanations related to Nest, you should search there.

Reviewer 2: Here are some suggestion to make some sentences more straightforward and to-the-point, with less repetitions (and a bit less frenglish):

```suggestion
The [introduction]() details generic but important concepts to grasp before being able to use Nest. The [chapter 2]() explains nest's commands while the [chapter 3]() explains some important concepts behind Nest. Although reading it is recommended, it is not required for a daily usage of nest (but it is for the usage of finest). The [chapter 4]() details the key concepts behind finest and its commands. In the [chapter 5](), you will find a cheat-sheet, with all nest's and finest's commands, and a short explanation on what they do. Finally, to conclude this guide, you will find a glossary in the [chapter 6](). It links all the commands, the important concepts and the technical terms mentioned in this guide to their respective chapters. If you're searching for explanations related to Nest, you should search there.
```

Reviewer: "generic but important concepts" ? I don't think "generic" is the word you're looking for.

Reviewer 2: Why not?

Author: `general`has the idea of being like overall concepts of Nest, without going too much into detail
whereas `generic` sounds like the concepts are related to a whole group or class, when it's only related to Nest. (Well ok there are `finest` and `nest`, but it feels weird to say `generic` instead of `general` for me when it's only about so few entities)

Reviewer 2: Ok! I'm fine with that reasoning! :)

# Getting started with Nest

## Foreword

> Welcome to the official guide to get you started with Nest! This guide assumes you are using the latest version of Nest, which is 0.1.0 at the time of writing.

Reviewer: ```suggestion
Welcome to Nest's official getting-started guide! This guide assumes you are using the latest version of Nest, which is 0.1.0 at the time of writing.
```

> In order to avoid any confusion, when we refer to `Nest` (with an upper case n), we refer to the project as a whole. When we refer to `nest` (with a lower case n), we only refer to the binary (executable file) nest. Thus, `finest` refer to its eponymous binary, and is included in Nest.

Reviewer: ```suggestion
In order to avoid any confusion, we will refer to the project as whole as `Nest` (with an upper case n) while we will refer to the binary as `nest` (with a lower case n). Thus, `finest` refer to its eponymous binary, and is included in Nest.
```
I feel like inverting both prepositions makes the sentence long-winded and confusing.

[//]: # (TODO: add link to the installation instructions page of Raven-OS)

> Nest is a dependency-based package manager for [Raven-OS](https://raven-os.org), which is an adaptive, customizable and evolutive open-source Operating System (OS) based on the Linux kernel. To use Nest, it's recommended to have installed Raven-OS to run Nest on it, because Nest may overlap with an already installed package manager. To install Raven-OS, follow the instructions [here](). Nest is included in the installation of Raven-OS by default, so you can start using it right away.

Reviewer: ```suggestion
Nest is a dependency-based package manager for [Raven-OS](https://raven-os.org), which is an adaptive, customizable and evolutive open-source Operating System (OS) based on the Linux kernel. To use Nest, it's recommended to install Raven-OS and to run Nest in it, because Nest may overlap with an already installed package manager. To install Raven-OS, follow the instructions [here](). Nest is included in the installation of Raven-OS by default, so you will be able to start using it right away.
```
Maybe we would even want to split that long sentence at because, because it is rather long-winded.

Author: there are five sentences in what you highlighted. Did you meant to say 'paragraph' instead of 'sentence'?

Reviewer: "To use Nest, it's recommended to install Raven-OS and to run Nest in it, because Nest may overlap with an already installed package manager."
Doesn't seem so bad now that I wrote it out.

## How to read this guide

> nest is the binary targeted towards newcomers. If this is the first time you are using Raven-OS, or more generally a GNU/Linux distribution, it's strongly recommended that you use this binary instead of finest. That latter is targeted towards more advanced users, and has more advanced commands. Both are installed by default with Nest, so you don't need to worry about installing them separately.

Reviewer: ```suggestion
nest is the binary targeted towards newcomers. If this is the first time you are using Raven-OS, or more generally a GNU/Linux distribution, it's strongly recommended that you use this binary instead of finest. The later is targeted towards more experimented users, and has more advanced commands. Both are installed by default with Nest, so you don't need to worry about installing them separately.
```
Proper English:
That latter -> The later
Better flow:
advanced users -> experimented

> nest is the binary targeted towards newcomers. If this is the first time you are using Raven-OS, or more generally a GNU/Linux distribution, it's strongly recommended that you use this binary instead of finest, as the latter is targeted towards more advanced users, with more advanced commands. Both are installed by default with Nest, so you don't need to worry about anything.

Reviewer: ```suggestion
nest is the binary targeted towards newcomers. If this is the first time you are using Raven-OS, or more generally a GNU/Linux distribution, it's strongly recommended that you use this binary instead of finest, as the latter is targeted towards more advanced users, and has more advanced commands. Both are installed by default with Nest, so you don't need to worry about installing them separately.
```
Otherwise, it's the users who have more advanced commands :P

[//]: # (TODO: Add link to the chapter 1)
[//]: # (TODO: Add link to the chapter 2)
[//]: # (TODO: Add link to the chapter 3)
[//]: # (TODO: Add link to the chapter 4)

> The [introduction]() details general but important concepts to understand to be able to use Nest.  The [chapter 2]() explains nest's commands. The [chapter 3]() explains some important concepts behind Nest. Although reading it is recommended, it's not required for the usage of nest (but it is for the usage of finest). The [chapter 4]() details the key concepts behind finest and its commands. In the [chapter 5](), you will find a cheat sheet, with all nest's and finest's commands, and a short explanation on what they do. Finally, to conclude this guide, you will find a glossary in the [chapter 6](), linking all the commands and the important concepts mentionned in this guide to their respective chapters. If you're searching for explanations related to Nest, you should go there.

Reviewer: ```suggestion
The [introduction]() details general but important concepts to understand to be able to use Nest.  The [chapter 2]() explains nest's commands. The [chapter 3]() explains some important concepts behind Nest. Although reading it is recommended, it's not required for the usage of nest (but it is for the usage of finest). The [chapter 4]() details the key concepts behind finest and its commands. In the [chapter 5](), you will find a cheat sheet, with all nest's and finest's commands, and a short explanation on what they do. Finally, to conclude this guide, you will find a glossary in the [chapter 6](), linking all the commands, technical terms and the important concepts mentioned in this guide to their respective chapters. If you're searching for explanations related to Nest, you should go there.
```
Also that last sentence sounds really weird to me.

Reviewer: Apparently the review system doesn't give detailed changes when comparing really long sentences. I fixed a grammar error and added "technical terms' to the list of things in the glossary.

Author: do you remember what that grammar error was? Since the review system doesn't want to show anything. I've added `technical terms` as you advised

## Getting help

> If you need help, you can join our official [communautary Discord server](https://invite.gg/ravenos) and ask any question you have there. Our team and community will be glad to answer them.

Reviewer 3: `Communautary` doesn't seem to be a real word, what you're looking for is probably `Communal`, but I think `Community` fits best here.
```suggestion
If you need help, you can join our official [community Discord server](https://invite.gg/ravenos) and ask any question you have there. Our team and community will be glad to answer them.
```

> If you're in need of help, you can join our official [communautary Discord server](https://invite.gg/ravenos) and ask any question you have there. Our team and community will be glad to answer them.

Reviewer: ```suggestion
If you need help, you can join our official [communautary Discord server](https://invite.gg/ravenos) and ask any question you have there. Our team and community will be glad to answer them.
```
